### PR TITLE
Replaced header image with real headings

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Replaced the image with the headings with actual headings so people can see how big it actually is.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

**Comment by Dusch4593** [comment]
@ethan-master-coding Very apt PR! This is super important considering the next section talks about accessibility. Not only will humans be able to "see" the headers, but so will screen readers hopefully.

**Comment by Dusch4593** [comment]
I'll take a look at this right now.

File: content/html/concepts/headings/headings.md [hunk 1]
```diff
@@ -22,8 +22,15 @@ Among the HTML elements are the 6 heading elements. They are labeled 1-6, with 1
...
-
-![Rendered page with the 6 HTML heading elements](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-headings.png)
+<details>
+<summary>Headings</summary>
+# Heading 1
+## Heading 2
+### Heading 3
+#### Heading 4
+##### Heading 5
+###### Heading 6
```
Comment (Dusch4593) [inline content/html/concepts/headings/headings.md:32]: I don't think this is rendering what the image was displaying? This worked for me, though: 

```html
<details>
<summary>Headings</summary>
<h1>Heading 1</h1>
<h2>Heading 2</h2>
<h3>Heading 3</h3>
<h4>Heading 4</h4>
<h5>Heading 5</h5>
<h6>Heading 6</h6>
</details>
```

Comment (ethan-master-coding) [reply content/html/concepts/headings/headings.md]: @Dusch4593 Got it. I made the change

**Review by Dusch4593 — APPROVED** [review]
LGTM!

**Comment by ethan-master-coding** [comment]
@Dusch4593 Can you look at what I said in the draft PR when you have a minute.

**Pull request merged**